doc/doxygen/src/driver-guide.md

```diff
@@ -166,45 +159,74 @@ CFLAGS="-DTMPABC_PARAM_ADDR=0x23" make all
 Second, we can override selected parameters from the board configuration
 (`board.h`):
 
-@code.{c}
-/* ... */
-/**
- * @brief   TMPABC sensor configuration
- * @{
- */
-#define TMPABC_PARAM_I2C        (I2C_DEV(1))
-#define TMPABC_PARAM_ADDR       (0x17)
-/** @} */
-/* ... */
-@endcode
+    /**
+    * @brief   TMPABC sensor configuration
+    * @{
+    */
+    #define TMPABC_PARAM_I2C        (I2C_DEV(1))
+    #define TMPABC_PARAM_ADDR       (0x17)
+    /** @} */
 
 Third, we can define more than a single device in the board configuration
 (`board.h`):
 
-@code{.c}
-/* ... */
-/**
- * @brief   Configure the on-board temperature sensors
- * @{
- */
-#define TMPABC_PARAMS           { \
-                                  .i2c  = I2C_DEV(1), \
-                                  .addr = 0x31 \
-                                }, \
-                                { \
-                                    .i2c  = I2C_DEV(1), \
-                                    .addr = 0x21 \
-                                }
-/** @} */
-/* ... */
-@endcode
+    /**
+    * @brief   Configure the on-board temperature sensors
+    * @{
+    */
+    #define TMPABC_PARAMS           { \
+                                      .i2c  = I2C_DEV(1), \
+                                      .addr = 0x31 \
+                                    }, \
+                                    { \
+                                        .i2c  = I2C_DEV(1), \
+                                        .addr = 0x21 \
+                                    }
+    /** @} */
 
 And finally, we can simply override the `tmpabc_params.h` file as described
 above.
 
+## Doxygen Documentation                                 {#driver-guide-doxygen}
+
+The above mentioned configurations can be made to show up in the listing for
+[Compile time configurations](@ref config). Refer to the following example that
+exposes TMPABC sensor to [sensors group](@ref config_drivers_sensors).
+
+    /**
+    * @brief   Default configuration parameters for TMPABC sensors
```

Comment:
I don't think it's necessary to copy again all the code here just to indicate that the macros can be grouped and added to the `config_drivers_sensors` group.

Also, this does not look like the best example of driver configuration, because you are adding the 'param' macros to the groups. We want to have the macros that configure the driver itself there (e.g. [TMP006](http://doc.riot-os.org/group__drivers__tmp00x__config.html))

Reply:
Oh. Shall replace it with a config from TMP00x them.